provider/resources.go

```diff
@@ -65,26 +65,27 @@ func Provider() tfbridge.ProviderInfo {
 		DisplayName:          "Materialize",
 		PreConfigureCallback: preConfigureCallback,
 		Config: map[string]*tfbridge.SchemaInfo{
-			"host": {
-				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_HOST"}},
-			},
-			"user": {
-				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_USER"}},
-			},
 			"password": {
 				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_PASSWORD"}},
 			},
-			"port": {
-				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_PORT"}},
-			},
 			"database": {
 				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_DATABASE"}},
 			},
+			"endpoint": {
```

Comment:
Update the provider level attributes. Specifically setting them so environment variable values can be set.